soc/nxp/imxrt/CMakeLists.txt

```diff
@@ -53,6 +63,7 @@ if(CONFIG_SOC_SERIES_IMXRT118X)
     PRIO 11
   )
   zephyr_compile_definitions(XIP_EXTERNAL_FLASH)
+  zephyr_compile_definitions(USE_HYPERRAM)
```

Comment:
should this be driven by a CONFIG?

Reply:
The `zephyr_compile_definitions(USE_HYPERRAM)` should be set at the board level, not SOC level. This C compiler definition is being used by our board specific file with XCMD setup commands to configure hyperram. From what I can tell, no file generic to the SOC depend on `USE_HYPERRAM` being defined, right? So a user could enable hyperram on a custom board without defining `USE_HYPERRAM`?

Reply:
Move `zephyr_compile_definitions(USE_HYPERRAM)` defined to board level.
Yes, `USE_HYPERRAM` is needed in SDK board driver if want to use hyperram.